cmd/gen.go

```diff
@@ -7,24 +7,25 @@ import (
 	"github.com/99designs/gqlgen/api"
 	"github.com/99designs/gqlgen/codegen/config"
 	"github.com/pkg/errors"
-	"github.com/urfave/cli"
+	"github.com/urfave/cli/v2"
 )
 
-var genCmd = cli.Command{
+var genCmd = &cli.Command{
 	Name:  "generate",
 	Usage: "generate a graphql server based on schema",
 	Flags: []cli.Flag{
-		cli.BoolFlag{Name: "verbose, v", Usage: "show logs"},
-		cli.StringFlag{Name: "config, c", Usage: "the config filename"},
+		&cli.BoolFlag{Name: "verbose, v", Usage: "show logs"},
+		&cli.StringFlag{Name: "config, c", Usage: "the config filename"},
 	},
-	Action: func(ctx *cli.Context) {
+	Action: func(ctx *cli.Context) error {
 		var cfg *config.Config
 		var err error
 		if configFilename := ctx.String("config"); configFilename != "" {
 			cfg, err = config.LoadConfig(configFilename)
 			if err != nil {
 				fmt.Fprintln(os.Stderr, err.Error())
 				os.Exit(1)
```

Comment:
return after exit doesn't do anything

Reply:
I know maybe I will try to remove os.Exit() and just return the err 
and use OnUsageError. 
```
OnUsageError: func(c *cli.Context, err error, isSubcommand bool) error {
          fmt.Fprintf(c.App.Writer, "for shame\n")
          return err
        },
```

Reply:
I think the exit and print is automatic when returning an error, maybe do a quick test?

`OnUsageError` is for when the arg parser fails I think?